.gitignore

```diff
@@ -2,3 +2,5 @@
 node_modules
 npm-debug.log
 docker-modem/
+.project
```

Comment:
Nah not at all. Just remove those lines, since you are at it please remove the line "docker-modem" too. And then just commit and push to your repo again, pr will update automatically.